parser/parse_test.go

```diff
@@ -253,6 +263,18 @@ const FooEnumSymbol proto.Enumeration = proto.EnumSymbol
 	if p := u.Package("base/foo/proto"); !p.HasImport("base/common/proto") {
 		t.Errorf("Unexpected lack of import line: %#v", p.Imports)
 	}
+	if len(u.Package("base/foo/proto").Constants) != 3 {
+		t.Errorf("Wanted 3 constants, got: %s", spew.Sdump(u.Package("base/foo/proto").Constants))
+	}
+	for k, v := range map[string]string{"EnumSymbol": `"enumSymbolValue"`, "TenBytes": "10", "ConstNumber": "99"} {
+		if c, ok := u.Package("base/foo/proto").Constants[k]; !ok {
+			t.Errorf("Missing constant %q", k)
+		} else {
+			if c.Constval == nil || *c.Constval != v {
+				t.Errorf("Unexpected value %#v for constant %q", v, k)
+			}
+		}
+	}
```

Comment:
optional nit: this could probably be simpler with `cmp.Diff(expectedConstants, u.Package("base/foo/proto").Constants))`....although `expectedConstants` would need to be `map[string]*types.Type` instead of `map[string]string`

Reply:
Good point; fixed!